test_gemini_simple.py

```
@@ -0,0 +1,69 @@
+import os
+import sys
+import json
+from dotenv import load_dotenv
+from prometheus_swarm.clients.gemini_client import GeminiClient
+
+def run_gemini_test():
+    # Try to load from .env if exists, but continue without it
+    try:
+        load_dotenv()
+    except:
+        pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)